README.md

```diff
@@ -147,12 +147,16 @@ tgpt -u
 ### Proxy
 
 Support:
+
+1. environment variable
```

Comment:
This isn't environment variable, its just the format of proxy

Reply:
I got it